.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16, 18]
+        node-version: [18]
```

Comment:
Should this include [Node 20](https://nodejs.org/en/blog/announcements/v20-release-announce) as well? (It's the Current version, will be LTS in October)

Reply:
+1, I think we can cover this change as part of https://github.com/react-native-community/cli/pull/1986

Reply:
https://github.com/react-native-community/cli/pull/2015